addons/mail/static/tests/chat_window/chat_window_tests.js

```diff
@@ -427,6 +427,36 @@ QUnit.test("open 3 different chat windows: not enough screen width", async (asse
     );
 });
 
+QUnit.test("close hidden chat window and check if there is still the right amount of visible chat windows", async (assert) => {
+    const pyEnv = await startServer();
+    pyEnv["discuss.channel"].create([
+        { name: "Channel_1" },
+        { name: "Channel_2" },
+        { name: "Channel_3" },
+        { name: "Channel_4" },
+        { name: "Channel_5" },
+    ]);
+    patchUiSize({ width: 900 });
```

Comment:
I would add the following assertions to make sure expected preconditions of tests are met:
```js
assert.ok(
        CHAT_WINDOW_END_GAP_WIDTH * 2 + CHAT_WINDOW_WIDTH * 2 + CHAT_WINDOW_INBETWEEN_WIDTH < 900,
        "should have enough space to open 2 chat windows simultaneously"
    );
    assert.ok(
        CHAT_WINDOW_END_GAP_WIDTH * 2 + CHAT_WINDOW_WIDTH * 3 + CHAT_WINDOW_INBETWEEN_WIDTH * 2 >
            900,
        "should not have enough space to open 3 chat windows simultaneously"
    );
```

If we change the sizing of chat windows, having these assertions makes life easier to adapt them ;)